packages/@aws-cdk/aws-lambda/test/test.vpc-lambda.ts

```diff
@@ -228,6 +244,58 @@ export = {
       });
     });
 
+    test.doesNotThrow(() => {
+      new lambda.Function(stack, 'PrivateLambda', {
+        code: new lambda.InlineCode('foo'),
+        handler: 'index.handler',
+        runtime: lambda.Runtime.NODEJS_10_X,
+        vpc,
+        vpcSubnets: { subnetType: ec2.SubnetType.PRIVATE },
+      });
+    });
+
+    test.doesNotThrow(() => {
+      new lambda.Function(stack, 'IsolatedLambda', {
+        code: new lambda.InlineCode('foo'),
+        handler: 'index.handler',
+        runtime: lambda.Runtime.NODEJS_10_X,
+        vpc,
+        vpcSubnets: { subnetType: ec2.SubnetType.ISOLATED },
+      });
+    });
```

Comment:
These assertions should be replaced with assertions that the CloudFormation template is correctly configured.

This will probably need to be split into separate tests for each subnet type.

Reply:
> These assertions should be replaced with assertions that the CloudFormation template is correctly configured.

You mean like a `hasResourceLike` assertion?

Reply:
Yes.